src/server/game/Entities/GameObject/GameObjectData.h

```diff
@@ -699,6 +699,7 @@ enum class GameObjectActions : uint32
     UseArtKit2,                     // Use ArtKit2
     UseArtKit3,                     // Use ArtKit3
     SetTapList,                     // Set Tap List
+    Max = SetTapList
```

Comment:
Why? the entire point of not assigning a value is to have it update automatically

Reply:
And how do you know which value is max from cpp ? Checking everything the header to make sure it's the latest ? And what if master branch gets more enum values and SetTapList is not the max anymore ? Will you check and update every place that uses SetTapList ?

Reply:
What I mean is not explicitly assigning a value to `Max`, not "not defining Max at all"

Reply:
ah ok